_sass/navigation.scss

```diff
@@ -274,6 +274,11 @@
   }
 }
 
+.main-menu #search-icon {
+  background-size: 14px 16px;
+  background-position: 6px 5px;
+}
+
```

Comment:
I think it makes more sense to edit these values in `search.scss` rather than add them here.